### PR TITLE
Register u.is-a.dev

### PR DESCRIPTION
**Pull request by ImShyMike**
Register u.is-a.dev with TXT record pointing to https://shymike.is-a.dev.

### CODE REVIEW

File: domains/u.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "ImShyMike",
+           "discord": "582648583635992622"
+        },
```
Comment (Stef-00012) [inline domains/u.json:5]: missing mail field

File: domains/u.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    
+        "record": {
+            "TXT": "https://shymike.is-a.dev"
```
Comment (Stef-00012) [inline domains/u.json:8]: TXT records alone do nothing, did you mean URL?

**Review by Stef-DP — DISMISSED** [review]

File: domains/u.json (hunk 2 above)
Comment (ImShyMike) [reply domains/u.json:8]: I did :kekw:

File: domains/u.json (hunk 1 above)
Comment (ImShyMike) [reply domains/u.json:5]: do I need to use the owl token?

Comment (CuteDog5695) [reply domains/u.json]: You just need to add the email field but leave it blank.

File: domains/u.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "ImShyMike",
+           "email": "",
+           "discord": "582648583635992622"
```
Comment (CuteDog5695) [inline domains/u.json:5]:
```suggestion
           "discord": "582648583635992622",
```

**Review by iostpa — DISMISSED** [review]

File: domains/u.json (hunk 3 above)
Comment (ImShyMike) [reply domains/u.json]: opsies

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**